### PR TITLE
Puppeteer: fixing .cookies() method type and other cleanup

### PR DESCRIPTION
**Pull request by davidkpiano**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pagecookiesurls>
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/puppeteer/index.d.ts [hunk 1]
```diff
@@ -38,29 +38,29 @@ export interface Dialog {
...
-  | "frameattached"
-  | "framedetached"
-  | "framenavigated"
-  | "load"
-  | "pageerror"
-  | "request"
-  | "requestfailed"
-  | "requestfinished"
-  | "response";
+  | 'console'
```
Comment (marvinhagemeister) [inline types/puppeteer/index.d.ts]: That's quite a lot of unnecessary diff noise here. Perhaps your editor automatically changed these lines?

File: types/puppeteer/index.d.ts [hunk 2]
```diff
@@ -300,7 +311,7 @@ export interface Page extends FrameBase {
   ): Promise<void>;
 
   // Argument `fn` can be an arbitrary function
-  exposeFunction(name: string, fn: any): Promise<void>;
+  exposeFunction(name: string, fn: Function): Promise<void>;
```
Comment (marvinhagemeister) [inline types/puppeteer/index.d.ts]: This will likely be caught be the `ban-types` linting rule.

File: types/puppeteer/index.d.ts [hunk 3]
```diff
@@ -291,7 +293,16 @@ export interface Page extends FrameBase {
   click(selector: string, options?: ClickOptions): Promise<void>;
   close(): Promise<void>;
   content(): Promise<string>;
-  cookies(...urls: string[]): Cookie;
+  cookies(...urls: string[]): Promise<Cookie[]>;
```
Comment (marvinhagemeister) [inline types/puppeteer/index.d.ts]: Good catch 👍

**Review by marvinhagemeister — CHANGES_REQUESTED** [review]
Good catch with the incorrect `cookies` method 👍  Remove the diff noise, change the `Function`  type (see comment) and this will be good to go in 💯

File: types/puppeteer/index.d.ts (hunk 1 above)
Comment (davidkpiano) [reply types/puppeteer/index.d.ts]: It did, I'll convert these back.

**Comment by andy-ms** [comment]
@davidkpiano Please undo the unnecessary quote changes and fix the lint failures.

**Comment by andy-ms** [comment]
Thanks!
